### PR TITLE
Add support for controlling the overrides of a Symbol

### PR DESCRIPTION
**Pull request by mathieudutour**
- [x] add `editable` property on `Override`
- [x] add `overrides` property on `Symbol Master`

### CODE REVIEW

File: Source/dom/layers/SymbolMaster.js [hunk 1]
```diff
@@ -120,3 +121,49 @@ SymbolMaster.define('symbolId', {
     throw new Error('Changing the symbol ID of a SymbolMaster is forbidden.')
   },
 })
+
+SymbolMaster.define('overrides', {
+  get() {
+    // undefined when immutable
+    if (!this._object.overrideProperies) {
```
Comment (bomberstudios) [inline Source/dom/layers/SymbolMaster.js]: Is this a typo on this code, or on Sketch?

Comment (bomberstudios) [reply Source/dom/layers/SymbolMaster.js]: (The `overrideProperies` bit)

Comment (mathieudutour) [reply Source/dom/layers/SymbolMaster.js]: in sketch. There is even `overrideProperiesByCombiningParentsOverrideProperties` haha

Comment (bomberstudios) [reply Source/dom/layers/SymbolMaster.js]: ¯\_(ツ)_/¯

**Pull request merged**